### DISCUSSION TITLE
How do I search by jurisdiction

### DISCUSSION THREAD

**Discussion by text2sql** [freelawproject/courtlistener#3114] [Q&A]
I understand I can filter by courts , but how do I go about filtering API to a specific jurisdiction, Illinois for example? Thank you

**Comment by mlissner** [comment]
There's no trick for that. You'll want to just grab all the IL court IDs and filter by all of them. The easiest way is to grab them from a search query URL, and copy/paste them into the API URL.

**Reply by legaltextai** [reply]
Thanks Mike. Do I take it that to download all Indiana decisions (e.g. court=ind) without running into a limit , I just break the download into chunks, with some time breaks in between the chunks?  Is it possible to remove that limit for a day?

**Reply by mlissner** [reply]
Well, yeah, you can use the API (provided you don't go too deep, and that you don't hit the speed limits). We can't lift it though, unfortunately, but what I usually recommend is to start with the bulk data and supplement with the API.
